### PR TITLE
feat(typescript_indexer): add references to imports of 3rd party modules

### PR DESCRIPTION
**Pull request by nbeloglazov**
Adds ref/imports edges from imports to modules declared in `.d.ts` files.

### CODE REVIEW

**Comment by nbeloglazov** [comment]
@evmar can you take a look? 

This PR depends on https://github.com/kythe/kythe/pull/4164 so it will fail until the other PR is merged.

File: kythe/typescript/indexer.ts [hunk 1]
```diff
@@ -1031,6 +1031,15 @@ class Visitor {
     if (modulePath) {
       const kModule = this.newVName('module', modulePath);
       this.emitEdge(this.newAnchor(moduleRef), EdgeKind.REF_IMPORTS, kModule);
+    } else {
+      // Check if module being imported is declared via `declare module`
+      // somewhere and if so - output ref to that statement.
+      const decl = moduleSym.valueDeclaration;
+      if (decl.kind === ts.SyntaxKind.ModuleDeclaration) {
```
Comment (evmar) [inline kythe/typescript/indexer.ts]: if (ts.isModuleDeclaration(decl)) {
and then below you don't need the 'as' coercion anymore

Comment (nbeloglazov) [reply kythe/typescript/indexer.ts]: Done.

**Comment by nbeloglazov** [comment]
@evmar friendly ping

**Pull request merged**